lang/java/reef-common/src/main/java/org/apache/reef/io/WorkingDirectoryTempFileCreator.java

```diff
@@ -39,8 +41,9 @@
   private final Path tempFolderAsPath;
 
   @Inject
-  WorkingDirectoryTempFileCreator() throws IOException {
-    this.tempFolderAsFile = new File("./reef/temp");
+  WorkingDirectoryTempFileCreator(
```

Comment:
It might be better to create a new `TempFileCreator` for this. The name `WorkingDirectoryTempFileCreator` suggests that it will always use the current working directory.